### PR TITLE
Update release notes draft to version v1.23.0-alpha.4

### PR DESCRIPTION
**Pull request by AuraSinis**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.23.0-alpha.4

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

File: releases/release-1.23/release-notes/maps/pr-101370-map.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pr: 101370
+releasenote:
+  text: Create HPA with follow：
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-101370-map.yaml:3]: Delete this file since it has no release notes from this PR.

File: releases/release-1.23/release-notes/maps/pr-101781-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 101781
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-101781-map.yaml:1]: Delete this file since it has no release notes from this PR.

File: releases/release-1.23/release-notes/maps/pr-103724-map.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 pr: 103724
 releasenote:
-  text: The image `gcr.io/kubernetes-e2e-test-images` will no longer be used in E2E
-    / CI testing, `k8s.gcr.io/e2e-test-images` will be used instead.
+  text: The image gcr.io/kubernetes-e2e-test-images will no longer be used in E2E
+    / CI testing, k8s.gcr.io/e2e-test-images will be used instead.
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-103724-map.yaml:4]: Revert the changes here since `gcr.io/kubernetes-e2e-test-images` and  `k8s.gcr.io/e2e-test-images` better be quoted.

File: releases/release-1.23/release-notes/maps/pr-104251-map.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 pr: 104251
 releasenote:
-  text: "Introduce `v1beta3` API for scheduler. This version \n- increases the weight
+  text: "Introduce v1beta3 API for scheduler. This version \n- increases the weight
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104251-map.yaml:3]: Revert the changes here.

File: releases/release-1.23/release-notes/maps/pr-104482-map.yaml [hunk 5]
```diff
@@ -1,4 +1,4 @@
 pr: 104482
 releasenote:
-  text: Changed kubectl describe to compute age of an event using the `EventSeries.count`
+  text: Changed `kubectl describe` to compute age of an event using the `EventSeries.count`
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104482-map.yaml:3]: Revert changes here

File: releases/release-1.23/release-notes/maps/pr-104631-map.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+pr: 104631
+releasenote:
+  text: A deprecation notice has been added when using the kube-proxy Userspace proxier,
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104631-map.yaml:3]: Userspace -> userspace

File: releases/release-1.23/release-notes/maps/pr-104817-map.yaml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 pr: 104817
 releasenote:
-  text: A pod that the Kubelet rejects was still considered as being accepted for
+  text: A Pod that the Kubelet rejects was still considered as being accepted for
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104817-map.yaml:3]: revert change back

File: releases/release-1.23/release-notes/maps/pr-104847-map.yaml [hunk 8]
```diff
@@ -1,4 +1,4 @@
 pr: 104847
 releasenote:
-  text: When a static pod file is deleted and recreated while using a fixed UID, the
-    pod was not properly restarted.
+  text: When a static Pod file is deleted and recreated while using a fixed UID, the
+    Pod was not properly restarted.
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104847-map.yaml:4]: revert changes back

File: releases/release-1.23/release-notes/maps/pr-104853-map.yaml [hunk 9]
```diff
@@ -0,0 +1,15 @@
+pr: 104853
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104853-map.yaml:1]: Delete this file since the PR does not have user face changes.

File: releases/release-1.23/release-notes/maps/pr-104877-map.yaml [hunk 10]
```diff
@@ -0,0 +1,4 @@
+pr: 104877
+releasenote:
+  text: Kubernetes object references (= name + namespace) were not logged as struct
+    when using JSON as log output format.
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104877-map.yaml:4]: Should be: Migrated kubernetes object references (= name + namespace) to structured logging.

File: releases/release-1.23/release-notes/maps/pr-105215-map.yaml [hunk 11]
```diff
@@ -1,3 +1,3 @@
 pr: 105215
 releasenote:
-  text: Kubelet's Graceful Nodee Shutdown will terminate probes when shutting down.
+  text: Kubelet's Graceful Node Shutdown will terminate probes when shutting down.
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-105215-map.yaml:3]: Kubelet's Node Grace Shutdown will terminate probes when shutting down.

File: releases/release-1.23/release-notes/maps/pr-105405-map.yaml [hunk 12]
```diff
@@ -0,0 +1,3 @@
+pr: 105405
+releasenote:
+  text: Ephemeral containers have reached beta maturity and are now available by default.
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-105405-map.yaml:3]: Change to: Ephemeral containers graduated to beta and are now available by default.

File: releases/release-1.23/release-notes/maps/pr-105474-map.yaml [hunk 13]
```diff
@@ -0,0 +1,4 @@
+pr: 105474
+releasenote:
+  text: Feature-gate VolumeSubpath has been deprecated and cannot be disabled. It
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-105474-map.yaml:3]: `VolumeSubpath`

File: releases/release-1.23/release-notes/maps/pr-105673-map.yaml [hunk 14]
```diff
@@ -0,0 +1,3 @@
+pr: 105673
+releasenote:
+  text: support more than 100 disk mounts on Windows
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-105673-map.yaml:3]: nit: Support

File: releases/release-1.23/release-notes/maps/pr-104877-map.yaml (hunk 10 above)
Comment (AuraSinis) [reply releases/release-1.23/release-notes/maps/pr-104877-map.yaml:4]: "Migrated kubernetes object references (= name + namespace) to structured logging when using Json as log output format" or just that?

Comment (cici37) [reply releases/release-1.23/release-notes/maps/pr-104877-map.yaml:4]: Let's go with "Migrated kubernetes object references (= name + namespace) to structured logging when using JSON as log output format". Thank you^

File: releases/release-1.23/release-notes/maps/pr-104251-map.yaml [hunk 15]
```diff
@@ -1,6 +1,6 @@
 pr: 104251
 releasenote:
-  text: "Introduce `v1beta3` API for scheduler. This version \n- increases the weight
+  text:"Introduce `v1beta3` API for scheduler. This version \n- increases the weight
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104251-map.yaml:3]: revert changes on this one

File: releases/release-1.23/release-notes/maps/pr-104915-map.yaml [hunk 16]
```diff
@@ -0,0 +1,4 @@
+pr: 104915
+releasenote:
+  text: Track the number of Pods with a Ready condition in Job status. The feature
+    is alpha and needs the feature gate JobReadyPods to be enabled.
```
Comment (cici37) [inline releases/release-1.23/release-notes/maps/pr-104915-map.yaml:4]: JobReadyPods -> `JobReadyPods`

**Comment by cici37** [comment]
Thank you for the awesome work! We should be ready to go after commits squashed.

**Comment by cici37** [comment]
When squash the commits, please keep the commit name as `Release Notes draft for k/k v1.23.0-alpha.4`. Thank you

**Comment by AuraSinis** [comment]
Found the issue:
- Did not git fetch --prune
- Generate and submit second draft pr with krel (with same name as first)
- Squash
- Rejected (stale info)
- Apparently, not actually rejected?
- Merge Conflicts between identical files
- k8s-ci-robot does not like merge commits

Solution:
- Revert to latest merge commit's oldest parent, then squashing further commits will work
